meta/recipes-connectivity/ppp/ppp_2.4.7.bb

```diff
@@ -86,6 +86,7 @@ do_install_append_libc-musl () {
 	install -Dm 0644 ${S}/include/net/ppp_defs.h ${D}${includedir}/net/ppp_defs.h
 }
 
+RDEPENDS_${PN} = "initscripts-functions"
```

Comment:
should this be `initscripts`

Reply:
ppp pnly requires only the "functions" file, which is part of initscripts-functions, I would rather not add additional dependencies, as this helps with creating minimal distribution

or is there something I'm missing?

Reply:
OK then it should be controlled by checking for sysvinit in DISTRO_FEATURES

Reply:
aha, seems like busybox and sysvinit will pull this in since they have VIRTUAL-RUNTIME_initscripts = "initscripts" which in turn rdepends on initd-functions which is provided by intscripts-functions

however in my case Im using toybox inittab and dont use initscripts at all, the idea of this patch is just to get that one dependency since its required by ppp run script

but thinking of that I guess this is a corner case and most users will just have initscripts on the platform

Reply:
right !!